.github/workflows/pr-request-review.yml

```diff
@@ -0,0 +1,49 @@
+name: Request Review
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  request-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if command is '/request_review'
+        id: check-command
+        run: |
+          if [[ $(echo "${{ github.event.comment.body }}" | tr '[:upper:]' '[:lower:]') == "/request_review"* ]]; then
+            echo "::set-output name=command::true"
+          else
+            echo "::set-output name=command::false"
+          fi
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Add label and request review
+        if: steps.check-command.outputs.command == 'true'
+        run: |
+          reviewers=$(echo "${{ github.event.comment.body }}" | grep -oE "@[a-zA-Z0-9_\-]+" | grep -v "@cryostatio/reviewer(s)" | paste -sd "," -)
```

Comment:
I think the reviewers team is called https://github.com/orgs/cryostatio/teams/reviewers ?